### ISSUE TITLE
Upload to App Center fails with error 400

### ISSUE THREAD

**Issue by swiatlow** [oliviergauthier/gradle-appcenter-plugin#72]
Hi, same issue is happening for me as described in closed issue [here](https://github.com/oliviergauthier/gradle-appcenter-plugin/issues/69)
Can't prepare release upload, code=400, reason={"status":400,"name":"Error","message":"Missing required body parameter \"body\""}

Is it maybe connected with the fact that endpoint is deprecated on AppCenter API?

**Comment by gmbett** [comment]
Did you try to upgrade to the latest version 1.3.0?

**Comment by swiatlow** [comment]
Yes, I'm using 1.3.0, tried older plugin version on different product but have the same result -  on both version the same deprecated url is being invoked for app upload (with 0 bytes body size when looking at logcat).

**Comment by gmbett** [comment]
Weird, version 1.3.0 works fine for me. I had indeed the same issue (missing body) with the previous version. As mentioned in the other ticket, the `file_size must be greater than 0` error can happen when the file that you are trying to upload doesn't exist.

**Comment by petersanta** [comment]
Hi @swiatlow I had a similar problem, when I used my `appCenterUpload` command as before (like `./gradlew clean appCenterUpload`).
(In my case `appCenterUpload` is responsible for uploading the apks.)

I recommend to call your gradle commands separately, something like this:
```
./gradlew clean
./gradlew assemble
./gradlew appCenterUpload
```
I hope this helps.
